apps/api/src/domain/models/audio-visual/audio-item/repositories/arango-audio-item-query-repository.ts

```diff
@@ -214,6 +214,41 @@ export class ArangoAudioItemQueryRepository implements IAudioItemQueryRepository
         await this.database.query({ query, bindVars });
     }
 
+    async importLineItems(id: AggregateId, lineItems: TranscriptLineItemDto[]) {
+        const query = `
+        FOR doc IN @@collectionName
+        FILTER doc._key == @id
+
+        update doc with {
+            transcript: MERGE(doc.transcript,{
+                items: APPEND(doc.transcript.items, @lineItems)
+            })
+        } IN @@collectionName
+        `;
+
+        const bindVars = {
+            '@collectionName': 'audioItem__VIEWS',
```

Comment:
Note that we will have to find a way to inject the collection name when we share this with the video query repository. That's a problem for later.